plonky2/src/gates/coset_interpolation.rs

```diff
@@ -933,16 +1014,58 @@ mod tests {
     #[test]
     fn test_num_wires_constraints() {
         const NUM_HASH_OUT_ELTS: usize = GOLDILOCKS_NUM_HASH_OUT_ELTS;
-        let gate = <CosetInterpolationGate<Goldilocks, 2>>::with_max_degree(4, 8);
-        assert_eq!(<CosetInterpolationGate<Goldilocks, 2> as Gate<Goldilocks, 2, NUM_HASH_OUT_ELTS>>::num_wires(&gate), 47);
-        assert_eq!(<CosetInterpolationGate<Goldilocks, 2> as Gate<Goldilocks, 2, NUM_HASH_OUT_ELTS>>::num_constraints(&gate), 12);
+        let gate = <CosetInterpolationGate<Goldilocks, GOLDILOCKS_EXTENSION_FIELD_DEGREE>>::with_max_degree(4, 8);
```

Comment:
For consistency, we should create a `const EXTENSION_FIELD_DEGREE` (or `D` for brevity) and reuse it in the test, in the same way we do for `NUM_HASH_OUT_ELTS` in the line above.

Reply:
Done